kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx

```diff
@@ -66,6 +66,17 @@ describe('Overview', () => {
     mockClearTopicMessages.mockClear();
   });
 
+  it('at least one replica was rendered', () => {
+    setupComponent({
+      ...defaultProps,
+      partitions: mockPartitions,
+      underReplicatedPartitions: 0,
+      inSyncReplicas: 1,
+      replicas: 2,
+    });
+    expect(screen.getAllByLabelText('replica-info')).toHaveLength(2);
```

Comment:
> at least one replica was rendered

But you expect 2

Reply:
Please check that you have correct values of Replica-info

Reply:
Fixed